### PR TITLE
Added UBERON:8410070 'levatores costarum' for #1904

### PR DESCRIPTION
**Pull request by paolaroncaglia**

### CODE REVIEW

**Comment by paolaroncaglia** [comment]
@dosumis could you please confirm that I've entered 
present_in_taxon "Homo sapiens"
in the correct way? The usage/syntax in Uberon seems inconsistent, I see both taxon labels and IDs being used. Thank you.

File: src/ontology/uberon-edit.obo [hunk 1]
```diff
@@ -225615,6 +225615,20 @@ relationship: in_taxon NCBITaxon:8782 ! Aves
...
+synonym: "levatores costarum muscle" EXACT []
+synonym: "musculi levatores costarum" EXACT LATIN []
+xref: FMA:71312
+is_a: UBERON:0002426 ! chest muscle
+relationship: has_muscle_insertion UBERON:0002228 ! rib
+relationship: has_muscle_origin UBERON:0001077 ! transverse process of vertebra
+property_value: http://purl.org/dc/elements/1.1/date 2022-01-27T10:41:44Z xsd:dateTime
+property_value: present_in_taxon "Homo sapiens" xsd:string
```
Comment (dosumis) [inline src/ontology/uberon-edit.obo:225629]: Should use NCBI_taxon IRI of Homo Sapiens as value.

**Comment by paolaroncaglia** [comment]
I'll merge for now; if I've entered present_in_taxon "Homo sapiens" incorrectly, I can fix it later, but it seems that it wouldn't be the only one in need of editing.

**Review by dosumis — CHANGES_REQUESTED** [review]

File: src/ontology/uberon-edit.obo (hunk 1 above)
Comment (paolaroncaglia) [reply src/ontology/uberon-edit.obo:225629]: Just saw this... will fix, but what about all the others?

**Comment by dosumis** [comment]
Oops - see you've merged.  We should now have a test for this.  I think all others have been fixed elsewhere now.

**Pull request merged**